### PR TITLE
[14.0][IMP][14.0]Add some optional fields to loan tree view

### PR DESCRIPTION
**Pull request by odooNextev**
Partner, pending_principal_amount and rate on loan Tree view as optional

### CODE REVIEW

**Review by francesco-ooops — APPROVED** [review]
Functional ok! minor improvement. 

@pedrobaeza would you mind a quick merge? thanks!

File: account_loan/static/description/index.html [hunk 1]
```diff
@@ -1,20 +1,20 @@
-<?xml version="1.0" encoding="utf-8"?>
+<?xml version="1.0" encoding="utf-8" ?>
```
Comment (pedrobaeza) [inline account_loan/static/description/index.html:1]: Why have you changed this file?

Comment (francesco-ooops) [reply account_loan/static/description/index.html:1]: I just asked for a readme rebuild as it was still linking to 12.0 runbot

image

Comment (odooNextev) [reply account_loan/static/description/index.html:1]: Hi Pedro, this change was made by oca-gen-addon-readme

**Pull request merged**